app/views/course/sections/school-placements.njk

```diff
@@ -23,7 +23,7 @@
   {% if schools.length %}
     <h3 class="govuk-heading-m">Locations</h3>
 
-    <p class="govuk-body">You can select one of the following locations when you apply.</p>
+    <p class="govuk-body">You can select one of the following locations when you apply to indicate your preferred placement.</p>
```

Comment:
Like this! Possibly it implies you’ll only have a single placement, when I think most trainees will do 2? But very hard to write this given how many different ways the locations list is used… 😩

Reply:
How about: You can select one of the following locations when you apply to indicate **a** preferred placement.